scripts/compatibility/split-cluster-check.sh

```diff
@@ -98,6 +98,20 @@ if ! grep -q "Node State has been reconfigured" "$LOG_DIR/fullnode.log"; then
   exit 1
 fi
 
+# ensure that the random beacon's DKG completes on both versions.
```

Comment:
fwiw i don't see how it could complete in one, but not the other, since it should cause a fork in that case.